src/data/arena-tag.ts

```diff
@@ -277,7 +277,7 @@ class AuroraVeilTag extends WeakenMoveScreenTag {
   }
 }
 
-type ProtectConditionFunc = (arena: Arena, moveId: MoveId) => boolean;
+export type ProtectConditionFunc = (moveId: MoveId) => boolean;
```

Comment:
When you start exporting it, consider moving it into it's own file inside `src/@types/`.

This ensures to keep our circular dep chances low